### ISSUE TITLE
particle offsets not working

### ISSUE THREAD

**Issue by DampfWaffel** [SkriptLang/Skript#2958]
Hello,
i've experienced some Bugs with the play/show effect

but it seems the when im discover one, im not able to Reproduce it every time.

**first**, the offset are not working. its spawn just a single point at the given loaction and the loading time for offsets are huge.
**second**, when i want more particles e.g `play 10 firework spark ...` it just do nothing or throw an error.
**third**, i think the endRod Particle is missing.

### Steps to Reproduce
first, `play firework spark with offset of 100, 40, 100 and speed 0 on player to player`
second, `play 10 firework spark on player to player`

### Errors / Screenshots
**first**, https://cdn.discordapp.com/attachments/645017130651222058/706107655160332388/7kRW5rT2OM.gif
**second**, https://pastebin.com/dkpSXzhs

### Server Information
* **Server version/platform:** Paper version git-Paper-161 (MC: 1.15.2)
* **Skript version:** Skript version 2.5-alpha3

Grats

**Comment by ShaneBeee** [comment]
Skript's effect for particles does not have an offset option:
http://skriptlang.github.io/Skript/effects.html#EffVisualEffect

My guess is you are using an add-on for this effect

**Comment by DampfWaffel** [comment]
ah that was quick, 
i found this offset at the SkriptHub Docs.
http://i.epvpimg.com/I3NXeab.png

https://github.com/SkriptLang/Skript/blob/14f267cc4bb8f00fa5c13536ce8a80afe64b1465/src/main/resources/lang/english.lang#L1464

and when i'm changing the speed it applies to the particle

**Comment by ShaneBeee** [comment]
That appears to be a mistake with their docs, because that is not written anywhere in Skript's code

**Comment by ShaneBeee** [comment]
Interesting, I didn't know that existed. 
I'll mark this as needs investigating.

**Comment by legofreak1999** [comment]
I can't test but it think this is not a bug but an syntax misperception.

There is a difference between visual effects and particles. 

In the syntax u write "on player" so skript thinks u want to play a visual effect from the entity player.
so like love hearts of a dog. But some effects are bound to specific entitys and a player has no effect of firework spark. So i think the parser parses the code for the location but doesn't register that its using now a location instead of an entity and since play on has no propertys for offset it doesn't work.

In short terms try this syntax:

`
play firework spark with offset of 100, 40, 100 and speed 0 at location of player to player
play 10 firework spark at location of player to player
`
